src/cc/export/helpers.h

```diff
@@ -588,5 +588,22 @@ int tracepoint__##category##__##event(struct tracepoint__##category##__##event *
             bpf_probe_read((void *)dst, __length, (char *)args + __offset); \
         } while (0);
 
+#define MEMBER_ADDR(src, field)                                             \
+        ({                                                                  \
+            void* __ret;                                                    \
+            __ret = (void*) (((char*)src) + offsetof(typeof(*src), field)); \
+            __ret;                                                          \
+        })
```

Comment:
Is this macro simpler? ({ (void *)&src->field }). If this is the case, do we really need this macro?